README.md

```diff
@@ -1,6 +1,6 @@
 # git-co-evg-base
 
-Find and checkout a recent git commit that matches the specified criteria.
```

Comment:
The command still allows user to perform git actions, so this change it wording seems incorrect. Maybe something like "Find and perform git actions on a recent commit..."

Reply:
Done